database/tables/questions.pg

```diff
@@ -2,6 +2,7 @@ columns
     client_files: text[] default ARRAY[]::text[]
     course_id: bigint not null
     deleted_at: timestamp with time zone
+    dependencies: jsonb default '{}'::jsonb
```

Comment:
Is the code able to handle `dependencies = NULL` or should we add a `NOT NULL` constraint here?

Reply:
No I don’t think it would, nice catch here.